apps/src/templates/MarketingSkinnyBanner.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
+import DCDO from '@cdo/apps/dcdo';
+
+// Not using inline styles to keep regular and responsive styles together.
+//   Copied styles from 'shared/css/csa_skinny_banner.scss' that's used
+//   in the Pegasus version of this banner:
+//  'pegasus/sites.v3/code.org/views/csa_skinny_banner.haml'.
+import './marketingSkinnyBanner.scss';
+
+const IMAGE_BASE_URL = '/blockly/media/';
+
+// Skinny banner component
+// Current: CSA Launch May 2022
+export default function MarketingSkinnyBanner() {
+  const header_text = 'A new approach to AP® CSA!';
```

Comment:
In javascript, variables should be named in camelcase so `headerText` (there are some places where we aren't consistent about this :( but we should be)

Reply:
Oh good catch! I copied it from the haml banner in pegasus but will update now